core/Tracker/Visitor.php

```diff
@@ -51,6 +51,35 @@ public function getVisitorColumn($column)
         return false;
     }
 
+    /**
+     * Get a visitor property that will not have been changed by other request processors
+     *
+     * @param string $column
+     *
+     * @return false|mixed|null
+     */
+    public function getImmutableVisitorColumn(string $column)
+    {
+        if (array_key_exists($column, $this->visitProperties->getImmutableProperties())) {
+            return $this->visitProperties->getImmutableProperty($column);
+        }
+
+        return false;
```

Comment:
Lets avoid using `false` as return value if the property can't be found. I know we are doing that in many places in Matomo, but that's actually not useful. A visit property can in theory hold the value `false`, which makes it impossible to rely on `false` as an indicator that the property doesn't exist. And if we ever have a case where we need to know if a property exists, we can simply create a new method for that purpose
```suggestion
        return $this->visitProperties->getImmutableProperty($column);
```